### ISSUE TITLE
Add a new Job source 

### ISSUE THREAD

**Issue by Pavel401** [Pavel401/Jobs-Scraper#3]
Hey there! 👋 We're always looking to expand our collection of job scrapers to provide a more comprehensive service. If you have a specific job board or website in mind that you'd like to see included we'd love your contribution!

### How to Contribute:

1. **Check for Existing Issues:**
   - Before starting, check if there's already an issue or pull request for the job board you have in mind.

2. **Fork the Repository:**
   - Fork the repository to your own GitHub account.

3. **Create a Branch:**
   - Create a new branch for your work to keep things organized.

4. **Write Code:**
   - Implement the new job scraper following the guidelines mentioned above.

5. **Test Your Code:**
   - Make sure your scraper works correctly and doesn't break existing functionality.

6. **Submit Pull Request:**
   - Submit a pull request with a clear title and description.

### Additional Information:

- Feel free to ask for help if you're facing any challenges.
- We appreciate your contribution and will review your pull request as soon as possible.

Let's work together to make our job scraper collection even better! 🚀

**Comment by praptisharma28** [comment]
Assign me @Pavel401

**Comment by Pavel401** [comment]
> Assign me @Pavel401

Create a new issue for the job source.
